compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -74,6 +74,21 @@ class ReifyQuotes extends MacroTransformWithImplicits {
 
   override def phaseName: String = "reifyQuotes"
 
+  override def checkPostCondition(tree: Tree)(implicit ctx: Context): Unit = {
+    if (ctx.owner.ownersIterator.exists(_.isTransparentMethod)) {
```

Comment:
Does it make sense to do these tests if `ctx.compilationUnit.containsQuotesOrSplices` is false?

Reply:
Yes, some other phase could inset by mistake one of those calls.